app/view_utils/manage_availability_helper.rb

```diff
@@ -0,0 +1,33 @@
+module ManageAvailabilityHelper
+
+  module_function
+
+  def availability_props(community:, listing:)
+
+    {
+      i18n: {
+        locale: I18n.locale,
+        defaultLocale: I18n.default_locale,
+        localeInfo: I18nHelper.locale_info(Sharetribe::AVAILABLE_LOCALES, I18n.locale)
+      },
+      marketplace: {
+        marketplace_color1: CommonStylesHelper.marketplace_colors(community)[:marketplace_color1],
+      },
+      listing: {
+        title: listing.title,
+        image_url: path_to_listing_image(listing),
+      }
+    }
+  end
+
+  def path_to_listing_image(listing)
+    if !listing.listing_images.empty?
+      image = listing.listing_images.first
+      if image.image_ready?
+        return image.image.url(:square)
+      end
+    end
+    nil
```

Comment:
There are some code style issues here (probably caught also by Rubocop):

1. Use either `unless listing.listing_images.empty?` or `if listing.listing_images.present?`
2. Remove `return` (line 27) and `nil` (line 30)

Or if you want to go zuper crazy:

```
Maybe(listings.listing_images.first)
  .select(&:image_ready?)
  .url(:square)
  .or_nil
```